projects/kit/styles/components/progress-bar.less

```
@@ -104,6 +104,18 @@
         border-radius: inherit;
     }
 
+    &[tuiProgressFixedGradient] {
+        .progressValue({
+            border-radius: inherit;
+
+            width: 100% !important;
+            margin-right: calc(-100% + var(--tui-progress-percent));
```

Good catch 👍 

Let's add comment add for descendants
```suggestion
            margin-right: calc(-100% + var(--tui-progress-percent)); // firefox
```